.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,99 @@
+name: Pre-release tests
+
+on:
+  # Runs at 5:00 UTC on Mondays
+  schedule:
+    - cron: "0 5 * * 1"
+  workflow_dispatch:
+
+jobs:
+
+  make-check-job:
+    name: Run 'make check'
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: pip install -r requirements-dev.txt
+      - name: Installing Avocado in develop mode
+        run: python3 setup.py develop --user
+      - name: Run make check
+        run: make check
+
+  avocado-pre-release-job:
```

Comment:
Since we are talking about jobs here, maybe we could remove the suffix "-job" on all of those.

Reply:
ok, I'll remove it 😄